source/modules/task-list.js

```diff
@@ -117,5 +203,38 @@
  * @description 
  */
 
+/**
+ * @name setSaveFlag
+ * @function
+ * @description helper function - set SAVE_FLAG for saving new task or editing task
+ * @param value SAVE_ON or EDIT_ON
+ */
+function setSaveFlag(value) {
+    SAVE_FLAG = value;
+}
+
+/**
+ * @name setTaskContent
+ * @function
+ * @description helper function - set TASK_CONTENT for editing task
+ * @param content is the task container
+ */
+function setTaskContent(content) {
+    TASK_CONTENT = content;
+}
+
+/**
+ * @name inputSanitizer
+ * @function
+ * @description helper function - set TASK_CONTENT for editing task
+ * @param content is the task container
+ */
+function inputSanitizer(input) {
+    if (input == "")
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9414198)
```suggestion
    if (input == '')
```